### PR TITLE
fixed minor issues in testgrid_flakiness_detection

### PR DESCRIPTION
**Pull request by oindrillac**
## Related Issues and Dependencies
Addresses #95 

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Fixed grammatical errors and remove code cells not being used in notebook.

### CODE REVIEW

File: notebooks/failure-type-classification/background/testgrid_flakiness_detection.ipynb [hunk 1]
```diff
@@ -21,17 +21,17 @@
    "execution_count": 1,
```
Comment (SankBad) [inline notebooks/failure-type-classification/background/testgrid_flakiness_detection.ipynb:21]: <p>Could you please run this block?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Comment (SankBad) [inline notebooks/failure-type-classification/background/testgrid_flakiness_detection.ipynb:21]: <p>Could you please run this block? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

**Comment by SankBad** [comment]
@oindrillac , 
Just added minor comments. Otherwise, LGTM

File: notebooks/failure-type-classification/background/testgrid_flakiness_detection.ipynb (hunk 1 above)
Comment (oindrillac) [reply notebooks/failure-type-classification/background/testgrid_flakiness_detection.ipynb:21]: @SankBad done :+1:

Comment (oindrillac) [reply notebooks/failure-type-classification/background/testgrid_flakiness_detection.ipynb:21]: @SankBad done :+1:

**Review by hemajv — APPROVED** [review]
LGTM :+1:

**Review by aakankshaduggal — APPROVED** [review]

**Review by SankBad — APPROVED** [review]
LGTM :train:

**Comment by MichaelClifford** [comment]
@aakankshaduggal for future reference please make sure the `aicoe-ci/build-check` and `aicoe-ci/pre-commit-check` are green before adding the "approved"  label as that will automatically merge the PR :smiley:  and we want to make sure the tests pass before merging.

**Comment by aakankshaduggal** [comment]
Sure @MichaelClifford will keep that in mind :+1:

**Pull request merged**